website/src/content/docs/docs/introduction/editor/vscode.md

```diff
@@ -75,18 +75,20 @@ The extension for Visual Studio Code collects usage data and sends it to Microso
 | -------------- | -------- | ---------------------------------- |
 | EventName      | string   | "start-extension" for example      |
 | ActivityId     | string   |                                    |
-| StartTime      | datatime |                                    |
-| EndTime        | datatime |                                    |
+| StartTime      | datetime |                                    |
```

Comment:
i think startTime and endTime is a bit no much information that is not really relevant, can we not just have the duration

Reply:
usually, we'd prefer to send raw data when possible so that later we can use kusto query to pivot data as needed. (i.e. use different timestamp to get different duration as needed) Otherwise, later when we want to analyze data from different perspectives, we may find we have to change code and wait for next release again just for the same data but in different form. thx